src/sass/Fabric.Icons.Output.scss

```diff
@@ -34,6 +34,10 @@
   @include ms-Icon--l;
 }
 
+// @todo: 'DynamicsCRMLogo' class is deprecated, will be removed in a future release
+// @todo: 'CarotRightSolid8' class is deprecated, will be removed in a future release
```

Comment:
It was a typo. The old class was 'carot' with an O, and the new is 'caret' with an E.